packages/dataviews/src/types.ts

```diff
@@ -123,6 +123,16 @@ export type Field< Item > = {
 	 */
 	isValid?: ( item: Item, context?: ValidationContext ) => boolean;
 
+	/**
+	 * Callback used to decide if a field should be displayed.
+	 */
+	isVisible?: ( item: Item ) => boolean;
```

Comment:
Both of these options may not make sense in the `DataViews` context? In which case we should create a specific `DataFormField` type.

Reply:
We have the same with `isValid` or `sort`. It's tempting to create separate "field types" for DataViews and DataForm but I don't think it's worth it right now. The consumers of this code will declare fields only once for both.